### DISCUSSION TITLE
Have you considered using google translate for (some) localizations?

### DISCUSSION THREAD

**Discussion by FuryFire2004** [zer0Kerbal/Dreamer#55] [Q&A] [answered]
self explanitory

**Comment by FuryFire2004** [comment]
I used it in French class when I took that like 4 years ago (too bad I don't remember any French)

**Reply by FuryFire2004** [reply]
do not use it for French though google wont tell you the feminine and masculine stuff that French language has,

**Reply by zer0Kerbal** [reply]
thankfully Watson does `Parlez-vous français`

**Reply by zer0Kerbal** [reply]
and in real life, from personal experience, most French people also speak English; often better than most native speakers. Just polite to at least start a conversation in French (Bonjour, Parlez-vous anglais ?) and of course it is only polite to use ( Merci beaucoup)

**Reply by zer0Kerbal** [reply]
same goes practically everywhere. Maybe not Alpha-Centauri or Mars. ;p

**Comment by zer0Kerbal** [comment, marked as answer]
**_quietly_** I have been using IBM Watson - and have almost gotten an auto translate program running with it. It will take `en-us.cfg` and spit out 12 additional translations files. only have so many characters allowed per month (twice what Google API allows) . I might try to use Google.Translate.API for the other 4-5 languages since it does translate many more.

**Reply by FuryFire2004** [reply]
thats actually really cool

**Reply by zer0Kerbal** [reply]
> thats actually really cool

thank you. it actually does more - it can read an entire mod's directory for parts, pull the title, description, manufacturer, and tags into en-us.cfg and replace them with auto-generated #LOC-strings. I also want it to eventually do that to MODULES like light, dock, etc (but that is for another day). It is truly the 🕳️ 🥤 of localization. that is also why its been and still is 😯 🤐 🙊

**Comment by AtomikkuSan** [comment]
I have some experience in Spanish and Japanese so when I do localizations via Google Translate, I usually check to see if it makes sense.

**Reply by zer0Kerbal** [reply]
exactly what I hope players do - because _'MI/AI isn't perfect, else it would be human.'_ 🤖
